pkgs/development/python-modules/osprofiler/default.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  netaddr,
+  oslo-concurrency,
+  oslo-serialization,
+  prettytable,
+  requests,
+  setuptools,
+  webob,
+}:
+
+buildPythonPackage rec {
+  pname = "osprofiler";
+  version = "4.1.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-V4N7YgWBsTREouIFZHddT8c+/2fHiIr5mUftWC/lFLA=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    netaddr
```

Comment:
We should add oslo-utils, too. In the next version we also need some others https://opendev.org/openstack/osprofiler/src/tag/4.1.0/requirements.txt